docs/contributing/database-migrations/edd.mdx

```diff
@@ -24,22 +23,72 @@ For background on this decision please see the [Evolutionary Database Design RFD
 
 ## Design
 
-### Nullable
+Martin Fowler's EDD defines two types of database changes: destructive and non-destructive
```

Comment:
🌱 With Fowler having largely named this, I'd like to suggest that we drop all the repeating of his name in this document and ensure links are good both inline and in the references section at the bottom.

Separately, I think we can also eliminate most statements of "EDD" too. This specific reference would simply be "Two types of database changes exist" ... and so on. EDD is implied throughout the document.

Reply:
I've removed all instances of Fowler's name in the article and am relying on the links at the bottom to give credit (with a few inline links to reference the `Further Reading` section)

I have also removed most of the "EDD" references as well.